applications/h264/install_dependencies.sh

```diff
@@ -38,12 +38,12 @@ gxf_mm_extensions=(["decoderio"]="45081ccb-982e-4946-96f9-0d684f2cfbd0" \
                    ["encoder"]="ea5c44e4-15db-4448-a3a6-f32004303338")
 mkdir -p gxf-mm
 mkdir -p ${HOLOSCAN_LIBS_DIR}
-CUDA_VERSION=12.2
+CUDA_VERSION=12.6
```

Comment:
With these changes, should we update the metadata.json to limit the `minimum_required_version` to `2.7`? Because if someone tries to build and run with images older than 2.7, it will not work anymore.

Reply:
Good catch, agreed, @apatole please update H264 `metadata.json` files accordingly